CHANGELOG.md

```diff
@@ -1,5 +1,53 @@
 # Optimizely Objective-C SDK Changelog
 
+## 3.0.0
+February 11, 2019
+
+The 3.0 release improves event tracking and supports additional audience targeting functionality.
```

Comment:
@aliabbasrizvi's nit from the Java PR: maybe include an empty line between each section? Not that it makes a difference in the rendered output.